package.json

```diff
@@ -12,6 +12,12 @@
     "lint-staged": "lint-staged",
     "prepare": "husky install"
   },
+  "lint-staged": {
+    "src/**/*.{js,jsx,ts,tsx,json,css}": [
+      "prettier --write",
+      "eslint --fix --max-warnings=0"
+    ]
+  },
```

Comment:
Dette gjøres i `.lintstagedrc.json`. Vi bør unngå å definere det to steder 😊

Reply:
Det skal være endret nå her [0d91d61](https://github.com/bekk/nav-familie-endringsmelding/pull/13/commits/0d91d6175262b4086bf085015979ef793d5c04a8)